### ISSUE TITLE
Auto conversion of /watch files fails: Cannot load libcuda.so.1

### ISSUE THREAD

**Issue by rjorgenson** [zocker-160/handbrake-nvenc-docker#11]
I have everything setup and tested the NVENC conversions in the GUI and they work just fine but when trying to use the auto conversion feature the encode fails with the below output in the conversion log

```[01:15:46] sync: expecting 322397 video frames
[01:15:46] encavcodecInit: H.265 (Nvidia NVENC)
[01:15:46] encavcodec: encoding at rc=vbr, multipass=fullres, 27.00
[01:15:46] encavcodec: encoding with stored aspect 1/1
[hevc_nvenc @ 0x1516642af900] Cannot load libcuda.so.1
[01:15:46] encavcodecInit: avcodec_open failed
ERROR: Failure to initialise thread 'FFMPEG encoder (libavcodec)'
[01:15:46] ac3-decoder done: 0 frames, 0 decoder errors
[01:15:46] hevc-decoder done: 0 frames, 0 decoder errors
[01:15:46] sync: got 0 frames, 322397 expected
[01:15:46] Finished work at: Sat Jun 25 01:15:46 2022

[01:15:46] libhb: work result = 3

Encode failed (error 3).
HandBrake has exited.
```

I loaded up a shell inside this container and I don't see the referenced `libcuda.so.1` file in this container anywhere. This is all very confusing because it all works just fine when encoding via the GUI. I am running the container in unRAID with a GTX 1050Ti.

```
# find / -iname 'libcuda.so.1'
find: '/proc/877/map_files': Permission denied
find: '/proc/931/map_files': Permission denied
find: '/proc/932/map_files': Permission denied
find: '/proc/983/map_files': Permission denied
find: '/proc/1019/map_files': Permission denied
find: '/proc/1157/map_files': Permission denied
find: '/proc/1367/map_files': Permission denied
find: '/proc/1368/map_files': Permission denied
find: '/proc/1369/map_files': Permission denied
find: '/proc/1370/map_files': Permission denied
find: '/proc/1372/map_files': Permission denied
find: '/proc/4074/map_files': Permission denied
find: '/sys/kernel/slab': Input/output error
#
```

**Comment by zocker-160** [comment]
thank you for this report, I will look into it

**Comment by zocker-160** [comment]
I sadly cannot reproduce this issue, it works for me using a 1080ti and the latest image from Docker Hub

![grafik](https://user-images.githubusercontent.com/36563538/175771094-d529b059-64be-4929-9c6e-860ceb349c58.png)

**Comment by rjorgenson** [comment]
I figured this out, as I expected it was user error. I did all the unRAID stuff to tell the container to use the GPU but I skipped the step that tells docker to _let_ it use the GPU, oops! I was wondering why it was working in the GUI and not in the terminal and it turns out that selecting the preset it in the GUI didn't break because it defaulted to the encoders that were available but specifying the preset from the command line did not. I thought it was working because it encoded but did not look closely enough, because it was definitely not. All is working now though!

In case anyone finds this using unRAID I had to pass `--runtime=nvidia` as an extra parameter to the container(this is the bit I missed) and set the `NVIDIA_VISIBLE_DEVICES` env variable to my GPU device ID. This was all that was required in addition to the env variables set in the docker file.
